requirements.txt

```diff
@@ -8,3 +8,5 @@ pydantic[email]==1.9.1
 vaderSentiment==3.3.2
 uvicorn[standard]==0.17.6
 gunicorn==20.1.0
+
```

Comment:
This is good work, let's get rid of that blank line (11) in the requirements.txt file.